src/integrationTest/java/uk/gov/hmcts/reform/cpo/repository/CasePaymentOrdersAuditTest.java

```diff
@@ -0,0 +1,157 @@
+package uk.gov.hmcts.reform.cpo.repository;
+
+import com.microsoft.applicationinsights.core.dependencies.apachecommons.lang3.RandomStringUtils;
+import org.hibernate.envers.AuditReader;
+import org.hibernate.envers.AuditReaderFactory;
+import org.hibernate.envers.RevisionType;
+import org.hibernate.envers.query.AuditEntity;
+import org.junit.ClassRule;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.testcontainers.containers.PostgreSQLContainer;
+import uk.gov.hmcts.reform.cpo.BaseTest;
+import uk.gov.hmcts.reform.cpo.data.CasePaymentOrderEntity;
+import uk.gov.hmcts.reform.cpo.domain.CasePaymentOrder;
+import uk.gov.hmcts.reform.cpo.service.mapper.CasePaymentOrderMapper;
+
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.NoResultException;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+import static uk.gov.hmcts.reform.cpo.repository.PostgresqlContainer.getInstance;
+
+@SuppressWarnings({"PMD.JUnitAssertionsShouldIncludeMessage", "PMD.DataflowAnomalyAnalysis"})
+public class CasePaymentOrdersAuditTest extends BaseTest {
+
+    @Autowired
+    private CasePaymentOrdersRepository casePaymentOrdersRepository;
+
+    @Autowired
+    private CasePaymentOrderMapper casePaymentOrderMapper;
+
+    @ClassRule
+    public PostgreSQLContainer postgreSQLContainer = getInstance();
```

Comment:
We can delete this

Reply:
👍

Reply:
Fixed in https://github.com/hmcts/cpo-case-payment-orders-api/pull/29/commits/997e6c96e81b2bb04894eeb6e64f23be1a43b5c9